config/q-expand.csv

```diff
@@ -457,10 +457,11 @@ QUEAA,Engineering
 QUEAAA,A*SYNC Cohort 
 QUEAAB,Stingarrays Cohort
 QUEAAC,Quacks Cohort 🦆🦆
-QUEAAD,The Cohort Formerly Known as Chumanjalaal
-QUEAAE,Unnamed Cohort E
+QUEAAD,Dingobots
+formerQUEAAD,Chumanjalaal
```

Comment:
Due to how the bot works, this likely won't show up with searches for QUEAA* searches. You could maybe do 
`QUEAAD,(former) Chumanjalaal`
Though I wonder do we really need to keep old ones? We haven't done it with other entries.

Reply:
I deliberately didn't want to have it show up in searches for `QUEAA*`.

We haven't done it with other old names, but Chumanjalaal is a bit of a special case because it's been around for so long, its origins are now lost to the mists of time, and the cohort went by “The Cohort Formerly Known as Chumanjalaal” for quite a while.